Assets/Vimeo/Scripts/Recorder/VimeoPublisher.cs

```diff
@@ -12,6 +12,9 @@ public class VimeoPublisher : MonoBehaviour
         public delegate void UploadAction(string status, float progress);
         public event UploadAction OnUploadProgress;
 
+        public delegate void RequestAction(string response);
```

Comment:
maybe name this `error_message` instead of `response`. response implies some sort of api json response

Reply:
https://github.com/vimeo/vimeo-unity-sdk/issues/29